src/utils/access/accessChecker.ts

```diff
@@ -0,0 +1,12 @@
+import { MODULE_CODES, MODULES } from "./modules.list";
+
+type actionType<T extends MODULE_CODES> = keyof typeof MODULES[T]["actionsAvailable"];
+
+/**
+ * @description Check whether loggedin user has access to the given module or not.
+ */
+export const UserHasAccess = <T extends MODULE_CODES>(moduleName: T, action: actionType<T>) => {
```

Comment:
Why does the function name starts with a capital U?

Reply:
only variable names should start from lowercase, Classes / function should start from Uppercase to distinguish them.

Reply:
A function name can be considered a variable. Never heard of functions starting with a capital letter, only classes or interfaces

Reply:
changed the methond name to camelCase